packages/atomic/src/components/search/result-template-components/atomic-result-image/atomic-result-image.tsx

```diff
@@ -36,27 +41,50 @@ export class AtomicResultImage implements InitializableComponent {
     return Array.isArray(value) ? value[0] : value;
   }
 
+  private handleImageError() {
+    let message =
+      'The image url is not valid. You might want to add a "fallback" property.';
+
+    if (this.fallback) {
+      message = 'The image url is not valid or could not be loaded.';
```

Comment:
Can we have the same first line in the msg in both cases and add `You might want to add a "fallback" property.'` conditionally?

Reply:
Can the invalid url be logged with the msg as well?

Reply:
Yes great idea ! I will change it.